content-src/components/ASRouterAdmin/ASRouterAdmin.jsx

```diff
@@ -740,5 +740,34 @@ export class ASRouterAdminInner extends React.PureComponent {
   }
 }
 
-export const _ASRouterAdmin = props => (<SimpleHashRouter><ASRouterAdminInner {...props} /></SimpleHashRouter>);
+export const CollapseToggle = props => {
+  const renderAdmin = props.location.hash && (props.location.hash.startsWith("#asrouter") || props.location.hash.startsWith("#devtools"));
+
+  const [collapsed, setCollapsed] = useState(false);
```

Comment:
I added a commit that reimplements the previous logic with `useState`/`useEffect`. What do you think @rlr?

Reply:
I reverted this for now, let's look at hooks later